libs/ui/dataviz/src/lib/table/table.component.html

```diff
@@ -32,9 +33,14 @@
         [class.active]="row.id === activeId"
       ></tr>
     </table>
-  </cdk-virtual-scroll-viewport>
-  <div class="text-gray-900 border-t border-gray-300 px-4 py-2 text-sm">
-    <span class="count font-extrabold text-primary">{{ count }}</span
-    >&nbsp;<span translate>table.object.count</span>.
```

Comment:
is this information gone?

Reply:
The count is now actually displayed in the pagination component.

Reply:
I've recovered it as discussed